src/Controller/KeyListBuilder.php

```diff
@@ -33,4 +33,14 @@ public function buildRow(EntityInterface $entity) {
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function render() {
+    $entities = $this->load();
+    $build = parent::render();
+    $build['table']['#empty'] = t('No keys are available. <a href="@link">Add a key</a>.', array('@link' => \Drupal::url('entity.key.add_form')));
```

Comment:
I modeled it after a similar message in core, so it should be correct, unless there are some very recent changes to Twig that would invalidate it. I agree that a test would be useful.